spec/controllers/users_controller_spec.rb

```diff
@@ -4,20 +4,20 @@
   include AuthSupport
 
   before(:each) do
-    usermgr_stub_auth
+    gdsuser_stub_auth
```

Comment:
this stops testing the user manager role... which is more important than GDS one... so I suggest adding more tests specifically for GDS users, instead of replacing the User Manager user

Reply:
We need to leave this until we bring in the the work to build the user from the JWT token as this work will make the user object team aware and populate the team within the user object.  Currently this fails as we can't access team off the user object.